mantidproject/mantid
# Indirect Data Analysis - Remove use of ADS in ProcessIndirectFitParameters

`ProcessIndirectFitParameters` is no longer reliant on the ADS during it's execution. Also minor updates in `CopyLogs` and `IqtFitSequential` and  ConvolutionFitSequential` have been made to accommodate for these changes.

**To test:**
- Code review of ProcessIndirectFitParameters changes to see the ADS is no longer used
- Ensure the following script executes without error (ensure the unit testing data directory is in the Data Search directories) :

**ProcessIndirectFitParameters**

```
# Create a host workspace
tws = WorkspaceFactory.createTable()
tws.addColumn("double", "A")
tws.addColumn("double", "B")
tws.addColumn("double", "B_Err")
tws.addColumn("double", "D")
tws.addRow([1,2,3,4])
tws.addRow([5,6,7,8])
tws.addRow([9,0,1,2])
tws.addRow([0,0,0,1])

# Add to Mantid Workspace list
mtd.addOrReplace("TableWs",tws)
wsName = "outputWorkspace"

# "D" is not included in the algorithm params list
ProcessIndirectFitParameters(tws, 'A', "B", OutputWorkspace=wsName)
```

Fixes #14064

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
